website/docs/guides/duckdb-qs.md

```diff
@@ -0,0 +1,255 @@
+---
+title: Quickstart for dbt Core using DuckDB
+id: duckdb
+description: "Learn to use dbt Core using DuckDB."
+hoverSnippet: "Learn to use dbt Core using DuckDB."
+platform: 'dbt-core'
+icon: 'duckdb-seeklogo'
+level: 'Beginner'
+hide_table_of_contents: true
+tags: ['dbt Core','Quickstart']
+---
+
+<div style={{maxWidth: '900px'}}>
+
+## Introduction
+
+In this quickstart guide, you'll learn how to use dbt Core with DuckDB, enabling you to get set up quickly and efficiently. [DuckDB](https://duckdb.org/) is an open-source database management system which is designed for analytical workloads. It is designed to provide fast and easy access to large datasets, making it well-suited for data analytics tasks. 
+
+
+This guide will demonstrate how to: 
+
+- Create a virtual development environment using a template provided by dbt Labs.
+   - This sets up a fully functional dbt environment with an operational and executable project. The codespace automatically connects to the DuckDB database and loads a year's worth of data from our fictional Jaffle Shop café, which sells food and beverages in several US cities.
+   - For additional information, refer to the [README](https://github.com/dbt-labs/jaffle_shop_duckdb/blob/duckdb/README.md) for the Jaffle Shop template. It includes instructions on how to do this, along with animated GIFs.
+- Run any dbt command from the environment’s terminal. 
+- Generate a larger dataset for the Jaffle Shop café (for example, five years of data instead of just one).
+
+You can learn more through high-quality [dbt Learn courses and workshops](https://learn.getdbt.com). 
+
+
+### Related content
+
+
+- [DuckDB setup](/docs/core/connect-data-platform/duckdb-setup)
+- [Create a GitHub repository](/guides/manual-install?step=2)
```

Comment:
the rest of these steps link to a diff quickstart, not sure if that's intentional?

Reply:
Hiya @mirnawong1 

Yep - this was intentional. The DuckDB one was to point users to the DuckDB setup page for general info on DuckDB and the GitHub one links users on the page in the DuckDB QS on how to create a repo (if they don't have one setup already.

I can remove these / change the links if you think it's best.

Kind Regards
Natalie